examples/stub.php

```diff
@@ -662,13 +662,13 @@ public function getOption(int $level, int $name) : int{}
 
     public function bind(string $host, int $port = 0) : bool{}
 
-    public function listen(int $backlog) : bool{}
+    public function listen(int $backlog = 0) : bool{}
 
     public function accept($class = self::class){}
 
     public function connect(string $host, int $port = 0) : bool{}
 
-    public static function select(array &$read, array &$write, array &$except, int $sec = 0, int $usec = 0, int &$error = null){}
+    public static function select(array &$read, array &$write, array &$except, int $sec, int $usec = 0, int &$error = null){}
```

Comment:
This should be `?int $sec`

Reply:
Good catch, do you see any other issues?

Reply:
Nothing off the hop, this is the only thing I noticed.